account.html

```diff
@@ -75,8 +75,7 @@ <h4 class="modal-title">My Account</h4>
 													<input type="text" class="form-control" id="newProjectTitle">
 													<br>
 													<label for="newProjectInfo" class="form-control-label">Project Description:</label>
-													<textarea class="form-control" id="newProjectInfo">
```

Comment:
fixed issue where the input field would start filled with a bunch of tabs
cleared tabs so input field starts empty for user